README.md

```diff
@@ -0,0 +1 @@
+[![🧪 CI/CD](https://github.com/kpi-web-guild/OlenaEfymenko/actions/workflows/ci.yml/badge.svg)](https://github.com/kpi-web-guild/django-girls-blog-OlenaEfymenko/actions/workflows/tox.yml?query=branch%3Amain)
```

Comment:
```suggestion
[![🧪 CI/CD](https://github.com/kpi-web-guild/OlenaEfymenko/actions/workflows/ci.yml/badge.svg)](https://github.com/kpi-web-guild/django-girls-blog-OlenaEfymenko/actions/workflows/ci.yml?query=branch%3Amain)
```

Reply:
Edited. Marking this conversation as resolved.